src/Arcus.Scripting.KeyVault/Scripts/Set-AzKeyVaultSecretFromFile.ps1

```diff
@@ -0,0 +1,15 @@
+# Use this script to upload a certificate as plain text (multiline-support) into Azure Key Vault.
+
+param (
+    [string][Parameter(Mandatory=$true)] $FilePath = $(throw "The path to the file is required."),
+    [string][Parameter(Mandatory=$true)] $KeyVaultName = $(throw "The path to the file is required."),
+    [string][Parameter(Mandatory=$true)] $SecretName = $(throw "The path to the file is required.")
+)
+
+Write-Host "Creating KeyVault secret..."
+
+$rawContent = Get-Content $FilePath -Raw
+$secretValue = ConvertTo-SecureString $rawContent -Force -AsPlainText
+Set-AzKeyVaultSecret -VaultName $KeyVaultName -SecretName $SecretName -SecretValue $secretValue -ErrorAction Stop
+
+Write-Host "Secret '$SecretName' has been created."
```

Comment:
Should we output the version of the new secret?

Reply:
Haven't required that yet, but I can see this would come in handy.
So, yes, let's include it.

Reply:
Hmm, apperently the version is not available in the returned PS model as part if the secret attributes.

Reply:
Why could include an additional call to retrieve the actual version of the uploaded secret:
`$secretVersion = (Get-AzKeyVaultSecret -VaultName $KeyVaultName -Name $SecretName).Version`

Reply:
Hmm, is that the worth the call? An extra API call just to log the secret version?

Reply:
From the looks of it, it should be in the return object:
https://docs.microsoft.com/en-us/powershell/module/az.keyvault/set-azkeyvaultsecret?view=azps-4.2.0#outputs

Reply:
See "Example 2: Modify the value of a secret using custom attributes"

Reply:
Hmm, that's not well documented then. And it isn't available as property but propably 'hidden' somewhere.
![image](https://user-images.githubusercontent.com/9039753/85383146-ee12d480-b53f-11ea-9ee5-983ff00d9956.png)

Reply:
I added the secret version of `.Attributes.Version` bc I think it's stored there. I couldn't be for sure.

Reply:
Ok, I changed that.